src/scripts/manage_artifacts/artifacts.py

```diff
@@ -1,78 +1,86 @@
+"""
+Manage cloudformation artifacts for a specific ref (namespace)
+"""
 import argparse
 import subprocess
 
-cfn_bucket = 'sceptre-cloudformation-bucket-bucket-65ci2qog5w6l'
-repo_name = 'BridgeDownstream'
+REPO_NAME = 'BridgeDownstream'
 
 
 def read_args():
-  descriptions = '''
-  Uploading to S3 and deletion from S3 of scripts and templates.
-  '''
   parser = argparse.ArgumentParser(
-    description='')
-  parser.add_argument(
-    '--ref')
+    description='Uploading to S3 and deletion from S3 of scripts and templates.',
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+  parser.add_argument('--ref')
```

Comment:
Nit: It might be helpful to add some `help` docs to these so that we can look back and know what each of these parameters are.

Reply:
I added some additional documentation.